pages/landing/hipaa/pricing/_price-col.html

```diff
@@ -14,7 +14,7 @@ <h2>Special early access pricing</h2>
       </div>
       <div data-mh="plans-pricing-bottom">
         <p>
-          <a id="pricing-cta-aws" class="btn btn-info pricing-ctas" href="/hipaa-compliance/contact" ga-on="click" ga-event-category="{{ page.path | slugify }}" ga-event-action="select-cta">I'm interested</a>
+          <a id="pricing-cta-aws" class="btn btn-info pricing-ctas" href="/hipaa-compliance-on-aws/contact" ga-on="click" ga-event-category="{{ page.path | slugify }}" ga-event-action="select-cta">I'm interested</a>
```

Comment:
```suggestion
          <a id="pricing-cta-aws" class="btn btn-info pricing-ctas" href="/hipaa-compliance-on-aws/contact" ga-on="click" ga-event-category="{{ page.path | slugify }}" ga-event-action="select-cta">Join the waitlist</a>
```

In the thread we talked about changing this to a stronger CTA, like the suggestion, or even just "Request Early Access". It looks like that hasn't actually happened yet. Do you want to sneak it in here?

Reply:
Sure, "request early access" sounds great, so I'll go with that.

Reply:
Actually, we've already got that as a link on the homepage, so I'll go with your suggestion.